sops/node-specific/GDI-SOP0002_NCPs-veto-EDIC-decision.md

```diff
@@ -0,0 +1,148 @@
+| Author     | Marcos Casado Barbero | Task 4.3 Lead | EMBL-EBI          |
+| Reviewer   | Regina Becker | LU / Pillar I co-lead | LNDS |
+| Approver   | _#! TO-DO_         |    |          |
+| Authorizer | _#! TO-DO_         |    |          |
+
+### 4. Purpose
+The purpose of this SOP is to delineate a standardised process for NCPs to review and potentially veto decisions made by the 1+MG EDIC regarding data access requests. This will ensure that data access is consistent with national regulations and ethical standards.
+
+### 5. Scope
+This SOP applies to all NCPs and Permit Authorities involved in the GDI project across participating countries, and relates to the 1+MG Data Governance (See [1+MG Data Governance for Research](https://docs.google.com/document/d/1P_nzGxMXG4CWzqkVbceY2fA3MQ7PEAwW)). It covers the procedures for reviewing and vetoing data access decisions made by the 1+MG DAC.
+
+### 6. Introduction and Background Information
+The GDI project aims to facilitate the integration and accessibility of genomic data across Europe, underpinned by the 1+MG initiative. The 1+MG EDIC is a central legal entity established through a Commission Implementing Decision. It orchestrates the infrastructure implementation of the multi-country project based on the 1+MG Declaration, central to overseeing data access requests. Given the strategic importance and sensitivity of genomic data, the role of NCPs and Permit Authorities are critical in ensuring that access decisions by the 1+MG DAC adhere to both national and European standards.
+
+### 7. Summary or Context Diagram
+
+![1+MG DAC, NCP, and Permit Authority processes for approval or veto of a 1+MG recommendation](../../docs/images/GDI-SOP0002_1-summary-diagram.jpg)
+**Figure 1**: 1+MG DAC, NCP, and Permit Authority processes for approval or veto of a 1+MG recommendation 
+
+### 8. Procedure
+#### 1. NCP Distribute the application recommendation
+| Step identifier | When                           | Who       |
+|:----------------|:-------------------------------|:----------|
+| 1               | Receipt of 1+MG DAC decision   | NCP       |
+
+Once the 1+MG DAC issues a decision, as the NCP, you should **distribute the application recommendation** to the appropriate 1+MG Data Holders and Permit Authority(ies), who will approve or deny the recommendation. Remind them to respond within 10 working days.
+
+Move to the following steps depending on the outcome of the step:
+- If a decision is communicated within 10 working days:
+   - If veto is not exercised, let the 1+MG know and move to **step 4**
+   - If **veto is exercised**, move to **step 2**.
+- If a decision is **not** communicated within 10 working days, move to step **step 4**.
+
+#### 2. Communicate justification of veto
+| Step identifier | When                           | Who       |
+|:----------------|:-------------------------------|:----------|
+| 2               | After step 1 and within 1 day  | NCP       |
```

Comment:
Perhaps "within 1 working day" instead of "within 1 day" (in line with text below)